.github/workflows/goreleaser-build-publish-develop.yml

```diff
@@ -25,6 +25,8 @@ jobs:
           role-to-assume: ${{ secrets.AWS_OIDC_IAM_ROLE_ARN }}
           role-duration-seconds: ${{ secrets.AWS_ROLE_DURATION_SECONDS }}
           aws-region: ${{ secrets.AWS_REGION }}
+          mask-aws-account-id: true
+          role-session-name: goreleaser-build-publish-chainlink.push-chainlink-develop-goreleaser
```

Comment:
I think this is causing errors because it is too long: https://github.com/smartcontractkit/chainlink/actions/runs/8708575878/job/23886281095